contracts/BridgeVault.sol

```diff
@@ -50,7 +49,9 @@ contract BridgeVault is Ownable, IBridgeVault {
         wETH.withdraw(amount);
 
         // Transfer the unwrapped ETH to the target address
-        targetAddress.transfer(amount);
+        // Send ether and specify a custom gas amount
+        (bool success, ) = targetAddress.call{value: amount/**, gas: 5000*/}("");
```

Comment:
there's no need to check for transfer success. it either succeeds or fails and reverts the TX. 

lets leave what we have before